packages/core/src/index.ts

```diff
@@ -241,7 +241,20 @@ function init(options: InitOptions): OnboardAPI {
   return API
 }
 
-function mountApp() {
+const fontFamilyExternallyDefined = (theme: Theme): boolean => {
+  if (!theme) return false
+  if (typeof theme === 'object' && theme['--w3o-font-family']) return true
+  if (
```

Comment:
Based on this logic, if you did not pass in a `theme` object, but defined a CSS var for the font, it would still return `false`?